src/System.Numerics.Matrices/System.Numerics.Matrices.csproj

```diff
@@ -1,9 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Similar to System.Numerics.Dec64, we are deleting System.Numerics.Matrices.

There is no steward/owner and the project has become inactive.

cc @KrzysztofCwalina, @whoisj 

It was added here: https://github.com/dotnet/corefxlab/pull/118